packages/fxa-settings/src/components/Settings/FlowRecoveryKeyInfo/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { LocationProvider } from '@reach/router';
+import FlowRecoveryKeyInfo from '.';
+import { Meta } from '@storybook/react';
+import { useFtlMsgResolver } from '../../../models';
+import { withLocalization } from '../../../../.storybook/decorators';
+
+export default {
+  title: 'Components/Settings/FlowRecoveryKeyInfo',
+  component: FlowRecoveryKeyInfo,
+  decorators: [withLocalization],
+} as Meta;
+
+export const Default = () => {
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedPageTitle = ftlMsgResolver.getMsg(
+    'test-page-title',
```

Comment:
Q: Should this be a valid FTL id? (I'm not sure if it matters if it's just a *.stories.tsx file)
```sh
  const localizedPageTitle = ftlMsgResolver.getMsg(
    'recovery-key-create-page-title',
    'Account Recovery Key'
  );
```

Reply:
I don't think it has to be but maybe I was wrong?

Reply:
You could use the existing string for this `recovery-key-create-page-title` as Peter suggested, that way it will actually be localized in the SB once we have those strings + would avoid errors with missing/mismatched id.